tests/alice.key

```diff
@@ -1 +0,0 @@
-b0d2d5dac3cc02d8db12e4a9703d6175cd4e9531027268b0566f50b85ca2e9410add5985d336d208fcb66c429f3abab7ae4d42f94714b54ea6fb8558d3a72df9
```

Comment:
I removed this file from source control by accident.  Was it supposed to be here?  `*.key` is listed in the top-level `.gitignore`.